src/Components/Caesar/Caesar.js

```diff
@@ -73,44 +78,52 @@ class Caesar extends Component {
                 <div className="container mb-6">
                     <div className="columns is-centered">
                         <div className="column">
+                        <Anime opacity={(this.props.startAnimation) ? fadein : [0,0]} translateY={bounce}>
                             <p className="is-size-4">
                                 The Year is 39 B.C., and Roman general Julius Caesar 
                                 wants to send a secret note to his friend Brutus.
                             </p>
+                            </Anime>
                         </div>
                     </div>
                     <div className="columns is-vcentered">
                         <div className="column is-one-quarter">
-                            <img src={caesar} alt="Caesar"/>
+                            <Anime opacity={(this.props.startAnimation) ? fadein : [0,0]} delay={2000}>
+                                <img src={caesar} alt="Caesar" id="caesar"/>
+                            </Anime>
                         </div>
                         <div className="column columns is-centered">
                             <div className="column is-four-fifths">
+                            <Anime opacity={(this.props.startAnimation) ? fadein : [0,0]} delay={5000}>

```

Comment:
I'm seeing a new bug around here. Now, when the user clicks on the message to decode it, it decodes for a split second before animating in the reverse direction.

Reply:
Reverse direction? I see it the same direction each time--but yes it does seem less intuitive now to have Brutus pop up for only a second after each click. Will work on clicking decodes one way each time.
So click -> encode
click again -> decode
click again -> encode